templates/accuracy_notice_email.html

```diff
@@ -0,0 +1,31 @@
+<p>
+    [Why me?] You are receiving this notification because you are listed as an owner of the following ChromeStatus feature entry:
+</p>
+</br>
+<p>
+    <strong>{{feature_name}}</strong>
+</p> 
+</br>
+<p>
+    [Why now?] Your feature is slated to launch soon:
+</p>
+</br>
+<p>
+    <strong>m{{milestone}}</strong>
```

Comment:
This was unclear in the design doc, but I was thinking that we should include all the feature milestones, like we do in an issue change notification.  So, it would be:
`{% include "estimated-milestones-table.html" %}`
I suppose the mstone that triggered this notification should me mentioned in a new paragraph or somewhere else in the email body.

Reply:
I understand. I've added this to add the table to the email.